examples/belief_propagation/bpsequences.jl

```diff
@@ -53,7 +53,7 @@ function main()
       contract_kwargs=(; alg="exact"),
       target_precision=1e-10,
       niters=100,
-      edges=edge_sequence(mts_init; alg=ITensorNetworks.Parallel()),
+      edges=edge_sequence(mts_init; alg=ITensors.Algorithm("Parallel")),
```

Comment:
You can add `using ITensors: Algorithm` to `src/imports.jl` to bring it `Algorithm` into the namespace.

Also you can just use `alg="parallel"` (note it should be lower case, everything should be lower case except types and modules).

Then we can wrap `alg` in `Algorithm` inside `edge_sequence`.